src/blocks/food-and-drinks/food-item/edit.js

```diff
@@ -4,6 +4,7 @@
 import { hasEmptyAttributes } from '../../../utils/block-helpers';
 import InspectorControls from './inspector';
 import icons from './icons';
+import fromEntries from '../../../js/coblocks-fromEntries';
```

Comment:
Just wanted to note what we had already talked about for future reference: We should let Babel handle transpiling rather than adding a polyfill ourselves manually. I looked into this and it seems core-js v3 package includes this polyfill but we are currently using core-js v2. We'll need to upgrade and resolve any issues sometime in the future.

We can leave this here for now.